source/docs/upgrade.md

```diff
@@ -30,6 +30,12 @@ The Logo component now depends on the [Laminas Text](https://github.com/laminas/
 
 Replace your `zendframework/zend-text` dependency with `"laminas/laminas-text": "^2.7"` in your `composer.json`.
 
+### Configuration changes
+
+The `app.production` configuration value has been removed and replaced by an `app.env` value in order to match Laravel.
+
+Replace `'production' => true,` with `'env' => 'development',` in your [`config/app.php`](https://github.com/laravel-zero/laravel-zero/blob/v7.0.0/config/app.php) file.
```

Comment:
Replace `'production' => true,` with `'env' => 'production',` in your [`config/app.php`](https://github.com/laravel-zero/laravel-zero/blob/v7.0.0/config/app.php) file.

Replace `'production' => false,` with `'env' => 'development',` in your [`config/app.php`](https://github.com/laravel-zero/laravel-zero/blob/v7.0.0/config/app.php) file.

Reply:
Thanks. 👍